Ghidra/Processors/CP1600/data/languages/CP1600.slaspec

```diff
@@ -0,0 +1,651 @@
+define endian=big;
+define alignment=2;
+define space ram type=ram_space wordsize=2 size=2 default;
+define space register type=register_space size=2;
+
+define register offset=0x00 size=2 [ R0 R1 R2 R3 R4 R5 R6 R7 ];
+define register offset=0x10 size=1 [ I D C O Z S ]; # Status bits
+define register offset=0x20 size=4 [ contextreg ];
+
+define token opcode_word (16)
+	clear              = (10, 15)
+	reg3_5             = (3, 5)
+	reg0_2             = (0, 2)
+	reg0_1             = (0, 1)
+	operation_size     = (2, 2)
+	
+	branch_sign        = (5, 5)
+	branch_external    = (4, 4)
+	branch_condition   = (0, 3)
+	external_condition = (0, 3)
+	
+	opcode6_9          = (6, 9)
+	opcode3_9          = (3, 9)
+	opcode2_9          = (2, 9)
+	opcode1_9          = (1, 9)
+	opcode0_9          = (0, 9)
+;
+
+define token jump_token (32)
+	unused_jump = (26, 31)
+	reg24_25    = (24, 25)
+	address_hi  = (18, 23)
+	jump_type   = (16, 17)
+	address_lo  = (0, 9)
+;
+
+define token double16 (32)
+	unused_lo = (17, 31)
+	value_lo  = (16, 23)
+	unused_hi = (8, 15)
+	value_hi  = (0, 7)
+;
+
+define token immediate16 (16)
+	imm16  = (0, 15)
+	addr16 = (0, 15)
+;
+
+define context contextreg
+	doublebyte = (0, 0)
```

Comment:
If you set this to noflow, you can remove all of the action blocks which set doublebyte to zero. Then you only have to worry about the action block for the `SDBD` instruction. This also circumvents the issue I highlighted on line 516.

Reply:
Maybe you're looking at an old version of the tree... That was already sorted out: https://github.com/agatti/ghidra/blob/cp1600/Ghidra/Processors/CP1600/data/languages/CP1600.slaspec#L48

Reply:
Weird, you're right. I checked the other day and it definitely wasn't showing up in the browser, but I can see the commit above and everything.